elixir/apps/api/lib/api/gateway/views/client.ex

```diff
@@ -1,6 +1,20 @@
 defmodule API.Gateway.Views.Client do
   alias Domain.Clients
 
+  def render(%Clients.Client{} = client, preshared_key) do
+    %{
+      id: client.id,
+      peer: %{
+        persistent_keepalive: 25,
+        public_key: client.public_key,
+        preshared_key: preshared_key,
+        ipv4: client.ipv4,
+        ipv6: client.ipv6
+      }
```

Comment:
While we are at it, can we flatten this? Would be easier to deserialise in Rust because we have to model less of these types.

Reply:
Done